docker-compose.yml

```diff
@@ -116,6 +116,9 @@ services:
             service: mender-base
         environment:
             - WORKFLOWS_MONGO_URL=mongodb://mongo-workflows:27017
+            - CREATE_ARTIFACT_GATEWAY_URL=https://mender-api-gateway
+            - CREATE_ARTIFACT_DEPLOYMENTS_URL=http://mender-deployments:8080
+            - CREATE_ARTIFACT_SKIPVERIFY=1
```

Comment:
What does this `SKIPVERIFY` do? It looks to me like something that should be in the `docker-compose-demo.yml` instead of here.

Reply:
@lluiscampos it is needed to skip verification of tls (aka, HTTPS) certificates when running the tests; you are right, it makes sense to move them to demo.